lib/Models/Internationalization.ts

```diff
@@ -148,16 +150,14 @@ class Internationalization {
 
           // cache user language on
           caches: ["localStorage"],
-          excludeCacheFor: ["cimode"], // languages to not persist (cookie, localStorage)
+          excludeCacheFor: ["cimode"] // languages to not persist (cookie, localStorage)
 
           // optional expire and domain for set cookie
           // cookieMinutes: 10,
           // cookieDomain: "myDomain",
-
-          // only detect languages that are in the whitelist
-          checkWhitelist: true
         },
         interpolation: {
+          skipOnVariables: false,
```

Comment:
Is this OK to set to false?
"If your interpolation variables are user provided or loaded from an external source, we strongly suggest to keep this option to true.
If you know what you're doing, you can also set this to false."
from https://www.i18next.com/translation-function/interpolation

Reply:
I see that this is the default behaviour pre v21, and we were previously using v19, so this is just retaining the same settings.

Reply:
Actually, we are safe to use skipOnVariables as true, I just went through i18next tests (only those cases that are documented won't be interpolated) and don't see it anywhere in the codebase. Thanks for raising this once again

Reply:
changed it to use a default value